package.json

```diff
@@ -209,6 +210,8 @@
 		"autoprefixer": "^9.8.0",
 		"babel-loader": "^8.1.0",
 		"bcrypto": "^5.3.0",
+		"classnames": "^2.2.6",
+		"codecov": "^3.8.1",
```

Comment:
@romanlikhovich just adding a reminder form our previous discussion. You could remove `codecov` if not necessary